coral-common/src/main/java/com/linkedin/coral/common/FuzzyUnionSqlRewriter.java

```diff
@@ -99,7 +101,7 @@ public SqlNode visit(SqlCall call) {
       // We need to compare the schemas of all leaves of the union (A,B,C) and adjust if necessary.
       // expectedDataType is the column subset of the union (A, B, C)
       final RelDataType expectedDataType = getUnionDataType(call);
-      call = addFuzzyUnionToUnionCall(call, expectedDataType);
+      addFuzzyUnionToUnionCall(call, expectedDataType);
```

Comment:
Can we actually change the signature for this method `addFuzzyUnionToUnionCall` ? I think having both side effects and a return type is not a good practice.

Reply:
Emm, that would be hard given we need it to return the call here:
https://github.com/linkedin/coral/blob/b515576e89cf14432a485ffc615ca9cb595ba4f6/coral-common/src/main/java/com/linkedin/coral/common/FuzzyUnionSqlRewriter.java#L190